engine/common/rpc/convert/convert.go

```diff
@@ -1212,6 +1185,43 @@ func MessageToTrieUpdate(m *entities.TrieUpdate) (*ledger.TrieUpdate, error) {
 	}, nil
 }
 
+// TrieUpdateToMessage converts a TrieUpdate to a protobuf message
+func TrieUpdateToMessage(t *ledger.TrieUpdate) (*entities.TrieUpdate, error) {
+	if t == nil {
+		return nil, nil
+	}
```

Comment:
This seems to be an error. 

Can the caller handle it outside of this function instead?

Reply:
this was because we don't want to return `&ledger.TrieUpdate{}`, it breaks the `ChunkExecutionDataToMessage` result with empty updates instead of `nil`